lib/pf/UnifiedApi/Controller/Nodes.pm

```diff
@@ -0,0 +1,65 @@
+package pf::UnifiedApi::Controller::Nodes;
+
+=head1 NAME
+
+pf::UnifiedApi::Controller::Nodes -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::Controller::Nodes
+
+=cut
+
+use strict;
+use warnings;
+use Mojo::Base 'pf::UnifiedApi::Controller::Crud';
+use pf::dal::locationlog;
+use pf::locationlog qw(locationlog_history_mac locationlog_view_open_mac);
+
+has dal => 'pf::dal::locationlog';
+has id_key => 'id';
+has resource_id => 'pid';
+
+sub latest_locationlog_by_mac {
+    my ($self) = @_;
+    my $mac = $self->param('mac');
+    $self->render(json => locationlog_view_open_mac($mac));
+}
+
+sub locationlog_by_mac {
```

Comment:
Missing the spec for this as well

Reply:
Added in https://github.com/inverse-inc/packetfence/pull/2907/commits/d3626bfd39fc1e8f0d9246d979ffe96dcab1b029